spyder/app/mainwindow.py

```diff
@@ -1854,6 +1873,43 @@ def trigger(i=index, self=self):
         else:
             self.ql_preferences.setEnabled(True)
 
+    def verifyMyMenu(self):

```

Comment:
@juanis2112 please change this to something like `verify_menu_actions`, also add a parameter for the list of menu actions something like `menu_actions` and use that parameter instead of running individually the search for actions in each list of menu actions. Something like this:

```python
def verify_menu_actions(self, menu_actions):
    for action in menu_actions:
        if isinstance(action, QAction):
            action.setIconVisibleInMenu(False)
```
Also don't forget to add a docstring :)